src/harness/reference_models/iap/iap.py

```diff
@@ -523,8 +523,8 @@ def performIapForFssCochannel(fss_record, sas_uut_fad_object, sas_th_fad_objects
   fss_low_freq, fss_high_freq = fss_freq_range
 
   # Get channels for co-channel CBSDs
-  fss_high_freq = min(fss_high_freq, interf.CBRS_HIGH_FREQ_HZ)
-  protection_channels = interf.getProtectedChannels(fss_low_freq, fss_high_freq)
+  assert fss_high_freq >= interf.CBRS_HIGH_FREQ_HZ, 'FSS high frequency should not be less than CBRS high frequency(Hz)'
```

Comment:
Are we sure that we want to use "assert" here, and not a traditional raise exception ?
assert can be completely disabled when launching your script with python -O or other option (optimize).
Why not= instead:
```
 if fss_high_freq >= interf.CBRS_HIGH_FREQ_HZ:
    raise ValueError('Fss High frequency should not be less than CBRS high frequency(Hz)')
```